### DISCUSSION TITLE
Cookie consent logging

### DISCUSSION THREAD

**Discussion by michael-van-den-berg** [orestbida/cookieconsent#616] [Q&A]
First at all thank you for all your hard work for this cookie consent script. It is really useful and really handy to use!

I have one question about the API in version 3, because I am not familiar with creating/hosting an API.

I would like to save the cookie preferences with an API to my own database (MySQL) so I have a ‘proof’ of consent. Therefor I need to create an API, as you described in the documentation of version 3.

What is the best way for creating an API? Is it possible to create one by my own and host it on my webserver? Or should I use an already existing one that I can use for the FETCH POST function.

I am using and embedded this cookie consent in a WordPress website. I am not sure if it is also possible by using the WordPress API and POST the data into a new table (for instance WP_cookieconsent)?

**Comment by charliebobgordon** [comment]
I'd be interested in this as well. It would be fantastic to have a completely open-source solution that also includes content logging and makes this part as user-friendly as the rest of the script. I'm unsure about how it could be implemented, though.
